File: Sigmath.ConvertedToVBNet/engineering/civil.vb
Public Class Civil
	Private Shared Function CalculateBeamDeflection(length As Double, load As Double, elasticity As Double, momentOfInertia As Double) As Double
		Return ((load * Basic.Power(length, 3)) / (3 * elasticity * momentOfInertia))
	End Function

	Private Shared Function CalculateStress(force As Double, area As Double) As Double
		Return (force / area)
	End Function

	Private Shared Function CalculateStrain(initialLength As Double, finalLength As Double, initialDiameter As Double) As Double
		Return ((finalLength - initialLength) / (initialLength * (initialDiameter / 2)))
	End Function

	Private Shared Function CalculateYoungsModulus(stress As Double, strain As Double) As Double
		Return (stress / strain)
	End Function
End Class

File: Sigmath.ConvertedToPython/engineering/civil.py
class Civil(object):
	def CalculateBeamDeflection(length, load, elasticity, momentOfInertia):
		return ((load * Basic.Power(length, 3)) / (3 * elasticity * momentOfInertia))

	CalculateBeamDeflection = staticmethod(CalculateBeamDeflection)

	def CalculateStress(force, area):
		return (force / area)

	CalculateStress = staticmethod(CalculateStress)

	def CalculateStrain(initialLength, finalLength, initialDiameter):
		return ((finalLength - initialLength) / (initialLength * (initialDiameter / 2)))

	CalculateStrain = staticmethod(CalculateStrain)

	def CalculateYoungsModulus(stress, strain):
		return (stress / strain)

	CalculateYoungsModulus = staticmethod(CalculateYoungsModulus)